src/main/java/com/jalasoft/webservice/entitities/OrcFile.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2019 Jalasoft.
+ *
+ *  This software is the confidential and proprietary information of Jalasoft.
+ *  ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
+
+package com.jalasoft.webservice.entitities;
+
+import com.jalasoft.webservice.utils.ConfigurationVariable;
+import net.sourceforge.tess4j.Tesseract;
+import net.sourceforge.tess4j.TesseractException;
+
+import java.io.*;
+
+/****
+ * OrcFile : Class to convert image to text
+ */
+public class OrcFile extends ModelFile {
+    public String getLang() {
+        return lang;
+    }
+
+    public void setLang(String lang) {
+        this.lang = lang;
+    }
+
+    private String lang;
```

Comment:
move this declaration before getter and setter methods

Reply:
done